apps/mobile/src/queries/balance/sip10-balance.query.ts

```diff
@@ -0,0 +1,37 @@
+import { QueryFunctionContext, useQuery } from '@tanstack/react-query';
+
+import { getSip10BalancesService } from '@leather.io/services';
+import { createMoney } from '@leather.io/utils';
+
+export function useSip10AggregateAvailableBalance(addresses: string[]) {
+  const { isLoading, data: balance } = useSip10AggregateBalanceQuery(addresses);
+  return !isLoading && balance ? balance.usd.availableBalance : createMoney(0, 'USD');
```

Comment:
Do you have any ideas what we can do to avoid defaulting to `createMoney(0, 'USD')`? 

Maybe if it has timed out and there is no balance we should set it to `createMoney(-1, 'USD')` / `createMoney(0, 'XXX')` so we can correctly pickup an error? (this isn't ideal but an idea) 

If we can't get their balance we shouldn't show a `0` balance and we [need to show an error state / fallback ](https://linear.app/leather-io/issue/LEA-505/loading-state-for-balances-on-home-screen)

Reply:
+1 what to do when handling loading/pending state is a concern of the UI. Defaulting to zero will lead to bugs and unexpected behaviours